x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/connector/connector_scheduling.tsx

```diff
@@ -61,7 +61,7 @@ export const ConnectorSchedulingComponent: React.FC = () => {
     frequency: schedulingInput?.interval ? cronToFrequency(schedulingInput.interval) : 'HOUR',
   });
 
-  if (!isConnectorIndex(index)) {
+  if (!isConnectorIndex(index, true)) {
```

Comment:
I'd rather we create a separate `isConnectorCrawler` check for this, rather than extending an existing function with a default parameter. The latter has a strong likelihood of creating more complexity, and (accidentally) missed checks.